docs/examples.rst

```diff
@@ -677,6 +677,34 @@ variables or context dimensions:
         ref: $@distro
 
 
+.. _anchors-aliases:
+
+Using anchors and aliases in Plans
+------------------------------------------------------------------
+
+When you need to specify the same variable multiple times,
+YAML has a feature called anchors and aliases. You can define an
+anchor before an item to save it for future usage with an alias.
+
+Example of an anchor:
+
+.. code-block:: yaml
+
+    discover:
+        how: fmf
+        test: &tests
```

Comment:
Different name perhaps? To make it more obvious it can be almost anything. For me "tests" is too close to tmt's attribute